docs-chef-io/content/inspec/profiles/depends.md

```diff
@@ -360,3 +362,44 @@ generates an `inspec.lock` file.
 
 If you add or update dependencies in `inspec.yml`, dependencies may be re-vendored
 and the lockfile updated with `inspec vendor --overwrite`
+
+## Gem based Resource Packs dependencies
+
+Any profile dependent on gem-based resource packs can specify them using the `gem` setting in the `inspec.yml` metadata file.
+
+InSpec handles the installation and download of the specified resource pack gems when running the profile.
+
+For example, if a profile uses a resource that has been deprecated and moved to a separate resource pack, the resource pack dependency can be specified in the metadata file as follows:
+
+```yaml
+depends:
+  - name: inspec-ibmdb2-resources
+    gem: inspec-ibmdb2-resources
+```
+
+where:
+
+- `name`: A logical identifier for the resource pack.
+- `gem`: The name of the Ruby gem that contains the resource pack.
+
+### Using Resource Packs of a Specific Version
+
+InSpec also allows specifying a particular version of a gem-based resource pack in the `inspec.yml` metadata file. This ensures that a specific version of the resource pack is used, providing stability and compatibility for the compliance tests.
+
+```yaml
+depends:
+  - name: inspec-ibmdb2-resources
+    gem: inspec-ibmdb2-resources
+    version: 7.0.1
+```
+
+### Using Resource Packs from a Specific Source
+
+InSpec also allows specifying a custom source for gem-based resource packs in the `inspec.yml` metadata file. This is useful when the resource pack is hosted on a private gem repository or an alternate source instead of the default RubyGems repository.
```

Comment:
Replace it with: InSpec also allows the `inspec.yml` metadata file to specify a particular version of a gem-based resource pack, ensuring its version provides stability and compatibility for compliance tests.

Reply:
This comment does not seem right since the above changes are for `sources` and not for `version`. Please check the review on this @dishanktiwari2501

Reply:
I have updated the comment.